src/lib.rs

```diff
@@ -27,7 +27,15 @@ fn connect(
     let rt = tokio::runtime::Runtime::new().unwrap();
     let db = match sync_url {
         Some(sync_url) => {
-            let fut = libsql::Database::open_with_remote_sync(database, sync_url, auth_token);
+            let ver = env!("CARGO_PKG_VERSION");
```

Comment:
@penberg we need to think about this, where do we want to pull the version number from?